Python/Anime-tracker/tracker.py

```diff
@@ -0,0 +1,66 @@
+
+try:
+    import requests
+    from bs4 import BeautifulSoup
+    import urllib.parse as parse
+    import re
+    
+
+except ImportError:
+    print('Some modules are not installed! ')
+
+# mainly bs4 lib is used for extracting html from web pages
```

Comment:
Use properly formatted comments. For eg: It can be replaced to `BS4 Library is being used to extract HTML from the Webpages`